sandbox/public/index.html

```diff
@@ -102,10 +103,30 @@
       // For Testing multiple instances.
       // We use a different orgId and edgeConfigId.
       organizationTwo("configure", {
-        edgeConfigId: "9999999",
+        edgeConfigId: "7984963a-6609-4e84-98d5-4e2ff8c0dd5e:prod",
         orgId: "97D1F3F459CE0AD80A495CBE@AdobeOrg",
         debugEnabled: true,
-        clickCollectionEnabled: false
+        clickCollectionEnabled: false,
+        onBeforeEventSend: function(options) {
+          const xdm = options.xdm;
+          const eventType = xdm.eventType;
+          if (eventType === "decisioning.propositionInteract") {
+            const analyticsPayloads = new Set();
+            const propositions = xdm._experience.decisioning.propositions;
+
+            for (var i = 0; i < propositions.length; i++) {
+              var proposition = propositions[i];
+              analyticsPayloads.add(getAnalyticsPayload(proposition));
+            }
+
+            getECID("organizationTwo").then(visitorID => {
+              const analyticsPayload = concatinateAnalyticsPayloads(
+                analyticsPayloads
+              );
+              triggerAnalyticsHit({ analyticsPayload, visitorID });
```

Comment:
Maybe call this something like sendAnalyticsPayload instead perhaps.

Reply:
ok, thank you

Reply:
Named it "triggerAnalyticsHit" so that is clear that this is the Analytics hit (like the one triggered by AppMeasurement), but contains the Target details as well.